test/integration/test-pipe.js

```diff
@@ -36,7 +37,7 @@ var server = http.createServer(function(req, res) {
       var field = FIELDS.shift();
       assert.strictEqual(name, field.name);
       assert.strictEqual(file.name, path.basename(field.value.path));
-      assert.strictEqual(file.type, mime.lookup(file.name));
+      assert.strictEqual(file.type, mime.lookup(file.name) || defaultContentType);
```

Comment:
And you already have test for default content type. I mean it would nice to make it as standalone test, not as part of custom-header test.